queries.sql

```diff
@@ -87,4 +87,38 @@ SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals
 
 SELECT species, AVG(escape_attempts) FROM animals
     WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31'
-    GROUP BY species;
\ No newline at end of file
+    GROUP BY species;
+
+
+-- What animals belong to Melody Pond?
+SELECT * FROM animals JOIN owners ON animals.owner_id = owners.id
+WHERE full_name = 'Melody Pond';
+
+-- List of all animals that are pokemon (their type is Pokemon).
+SELECT * FROM animals JOIN species ON animals.species_id = species.id
+WHERE species.name = 'Pokemon';
+
+-- List all owners and their animals, remember to include those that don't own any animal.
+SELECT * FROM animals RIGHT JOIN owners ON animals.owner_id = owners.id;
+
+-- How many animals are there per species?
+SELECT species.name, COUNT(*) from species 
+left join animals on species.id = animals.species_id
+group by species.name;
+
+-- List all Digimon owned by Jennifer Orwell.
+SELECT * FROM owners 
+JOIN animals ON owners.id = animals.owner_id
+JOIN species ON species.id = animals.species_id
+WHERE owners.full_name = 'Jennifer Orwell';
```

Comment:
- [ ] @quavo19 great job on your query you forgot to add code to specify animal species being listed eg `WHERE owners.full_name = 'Dean Winchester' AND species.name = 'Digimon';`

'don't forget to update the screenshot query result to reflect the changes`

Reply:
@danmainah I don't think that's part 
there are seven queries and I wrote them all. for Dean Winchester, the query is at line 116

Reply:
@danmainah  please which part of the requirement requires this `owners.full_name = 'Dean Winchester' AND species.name = 'Digimon';`

Reply:
- [ ] @quavo19 you misunderstood that is just an example query, the requirement was you list the animal owned by Jennifer Orwell from the Digimon species, your query only showed animals owned by Jennifer Orwell.